database/postgres/ddl/build.go

```diff
@@ -61,5 +61,11 @@ CREATE INDEX
 IF NOT EXISTS
 builds_status
 ON builds (status);
+`
```

Comment:
super picky but for consistency, can we add blank line after this and add a comment before the new query?